### PR TITLE
fix(components): fix Aside tabs

### PR DESCRIPTION
**Pull request by eduardnicolae30**
fix(components): fix Aside tabs

The current implementation for rendering aside tabs doesn't work due to the fact that the div that should incorporate them is missing.

Screenshot 2020-07-07 at 16 30 13

### CODE REVIEW

**Comment by eduardnicolae30** [comment]
npm publish

**Comment by n-srg** [comment]
@wejendorp who can help us with this PR? maybe the best and the only.... you? :) 

the reference is there http://ui.tradeshift.com/v12/#components/asides/

File: src/components/Aside/index.js [hunk 1]
```diff
@@ -2,13 +2,13 @@ import React, { Component } from 'react';
...
 
-const noop = () => {};
+const noop = () => { };
 
 class Aside extends Component {
 	static wrapContent(nodes) {
 		const children = React.Children.toArray(nodes);
 		const wrappedTabs = children.every(node => node.props['data-ts'] === 'Panel');
-		return nodes && wrappedTabs ? nodes : <div data-ts="Panel">{nodes}</div>;
+		return nodes && wrappedTabs ? children.length === 1 ? nodes : <div data-ts="Panels">{nodes}</div> : <div data-ts="Panel">{nodes}</div>;
```
Comment (wejendorp) [inline src/components/Aside/index.js:11]: Can we split this into not using a nested ternary if? They're kinda hard to read.

Maybe make it two separate returns instead.

File: src/stories/__tests__/__snapshots__/Storybook.spec.js.snap [hunk 2]
```diff
@@ -9,16 +9,20 @@ exports[`Aside Aside tabs 1`] = `
     data-ts.title="Aside demo"
   >
     <div
-      data-ts="Panel"
-      data-ts.label="tab 0"
+      data-ts="Panels"
```
Comment (wejendorp) [inline src/stories/__tests__/__snapshots__/Storybook.spec.js.snap:12]: Is the test still fully covering the new branching?

Comment (eduardnicolae30) [reply src/stories/__tests__/__snapshots__/Storybook.spec.js.snap:12]: This test covers the case when the aside has tabs

**Pull request merged**